libexec/bats

```diff
@@ -100,34 +100,38 @@ for arg in "$@"; do
 done
 
 unset count_flag pretty
+count_flag=''
```

Comment:
As we don't have a styleguide... is there a strong objection to using `""` for all the variable (re)sets?

Reply:
My general approach is single quotes for anything that doesn't have a variable expansion or a `'` character. But it isn't a strong guideline.